x-pack/plugins/endpoint/public/applications/endpoint/store/policy_list/reducer.ts

```diff
@@ -46,3 +63,25 @@ export const policyListReducer: Reducer<PolicyListState, AppAction> = (
 
   return state;
 };
+
+export const policyDetailsReducer: Reducer<PolicyDetailsState, AppAction> = (
+  state = initialPolicyDetailsState(),
+  action
+) => {
+  if (action.type === 'serverReturnedPolicyDetailsData') {
+    return {
+      ...state,
+      ...action.payload,
+      isLoading: false,
+    };
+  }
+
+  if (action.type === 'userChangedUrl') {
```

Comment:
It's also probably time to break this cycle and put location in a more generic place in state